### PR TITLE
Remove hidden but usable CLI arguments

### PR DESCRIPTION
**Pull request by Kerollmops**
@curquiza found out that we were exposing some internal CLI arguments: `nb-max-chunks` and `log-every-n`. In this PR I removed those two, the only two ones that I found. Those options shouldn't be accessible as non-documented in the documentation or the `--help` message.

Fixes https://github.com/meilisearch/meilisearch/issues/3307

### CODE REVIEW

File: meilisearch/src/option.rs [hunk 1]
```diff
@@ -63,7 +63,7 @@ const DEFAULT_DUMP_DIR: &str = "dumps/";
 
 const MEILI_MAX_INDEXING_MEMORY: &str = "MEILI_MAX_INDEXING_MEMORY";
 const MEILI_MAX_INDEXING_THREADS: &str = "MEILI_MAX_INDEXING_THREADS";
-const DEFAULT_LOG_EVERY_N: usize = 100000;
+const DEFAULT_LOG_EVERY_N: usize = 100_000;
```
Comment (dureuill) [inline meilisearch/src/option.rs:66]: I feel like we could remove this, as it doesn't appear to be used anywhere now. What do you think?

**Review by dureuill — CHANGES_REQUESTED** [review]
The PR looks good, I have one question (see inline comment).

File: meilisearch/src/option.rs (hunk 1 above)
Comment (Kerollmops) [reply meilisearch/src/option.rs:66]: It is used by milli in the indexer part to log for indexed documents.

**Review by dureuill — APPROVED** [review]
Found the `DEFAULT_LOG_EVERY_N` 😅

**Comment by Kerollmops** [comment]
bors merge

**Pull request merged**